ldcbench.api/src/main/java/org/dice_research/ldcbench/rdf/SimpleCachingTripleCreator.java

```diff
@@ -35,12 +35,12 @@ public SimpleCachingTripleCreator(int baseGraphId, String[] resourceUriTemplates
     }
 
     @Override
-    public Node createNode(int nodeId, int externalId, int extGraphId, boolean isProperty) {
+    public Node createNode(int nodeId, int externalId, int extGraphId, boolean isProperty, boolean isBlankNode, boolean isLiteral) {
```

Comment:
Now, that we have a lot of these flags, the usage of an `enum` could be more elegant, or? Something like
```java
public enum RDFNodeType {
    IRI,
    Property,
    BlankNode,
    Literal
}
```
What do you think? It would make these methods easier although it may complicate the ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/utils/TripleIterator.java class. :thinking: